core/kernel/tee_ta_manager.c

```diff
@@ -854,6 +854,13 @@ static TEE_Result dump_ta_memstats(struct tee_ta_session *s,
 	if (!ts_ctx)
 		return TEE_ERROR_ITEM_NOT_FOUND;
 
+	/*
+	 * if the user TA is still being initialized,
```

Comment:
(nit) s/if/If/
maybe even remove the inline comment: the implementation is explicit.

Reply:
thanks for suggestion. done